apps/web/components/ui/BlockActions/BlockActions.vue

```diff
@@ -11,16 +11,27 @@
       </SfIconBase>
     </button>
 
-    <!-- Re add in when we reach the sorting PBI's Re. Add imports as well-->
+    <div class="w-px h-4 bg-gray-300" />
 
-    <!-- <div class="w-px h-4 bg-gray-300"></div>
-    <button class="text-black hover:bg-gray-100 p-1 rounded" data-testid="move-down-button">
+    <button
+      class="text-black hover:bg-gray-100 p-1 rounded"
+      data-testid="move-up-button"
```

Comment:
```suggestion
      :data-testid="'move-up-button-${props.index}'"
```
This way you can access a specific block's button in the test.

Reply:
I undestand the concept but I think in our case it is not suited.
If I will go this way, I will need to have 3 lines of code for every button I want to select:

So instead of having this: `  get bottomMoveBlockButton() {
    return cy.getByTestId('move-down-button')
  }`
  
  I will be having this: `  get bottomMoveBlock1Button() {
    return cy.getByTestId('move-down-button-1')
  }
  
    get bottomMoveBlock2Button() {
    return cy.getByTestId('move-down-button-2')
  }
  `